packages/moleculer-db-adapter-mongo/src/index.js

```diff
@@ -209,7 +210,10 @@ class MongoDbAdapter {
 	 * @memberof MongoDbAdapter
 	 */
 	updateById(_id, update) {
-		return this.collection.findOneAndUpdate({ _id: this.stringToObjectID(_id) }, update, { returnOriginal : false }).then(res => res.value);
```

Comment:
What's the reason for changing `findOneAndUpdate` and `findOneAndDelete`

Reply:
Yes you're right, I thought that the key "value" in the returned object did not exists anymore. But it still exists.
So I just put the methods back.